atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnRangeSelection.java

```diff
@@ -84,4 +89,8 @@ public String toString() {
         String end = PtBytes.encodeBase64String(endCol);
         return Joiner.on(',').join(ImmutableList.of(start, end));
     }
+
+    private static boolean isValidRange(byte[] startCol, byte[] endCol) {
```

Comment:
Can you expose this as a method, same as in the RangeRequest? Or delete it from both and then I'll happily use RangeRequests.isValidRange directly.

Reply:
Will do. It turns out the logic here is not equivalent (if you see @gmaretic's comment below)

Reply:
so I'll expose this, and we should probably stick with that.